Magento_Company/templates/company/management/profile/edit.phtml

```diff
@@ -0,0 +1,258 @@
+                    class="input-text input__field"
+                    data-validate="{ required: true }"
+                >
+            </div>
+            <div class="dashboard__input input street">
+                <input
+                    type="text"
+                    name="street[1]"
+                    id="street1"
+                    value="<?= (!empty($street[1])) ? $block->escapeHtml($street[1]) : ''; ?>"
+                    title="<?= $block->escapeHtml(__('Street Address')) ?>"
+                    class="input-text input__field"
+                >
+            </div>
+            <div class="dashboard__input input city _required">
+                <label class="label" for="city">
+                    <?= $block->escapeHtml(__('City')) ?>
+                </label>
+                <input
+                    type="text"
+                    name="city"
+                    id="city"
+                    value="<?= $block->escapeHtml($company->getCity()) ?>"
+                    title="<?= $block->escapeHtml(__('City')) ?>"
+                    class="input-text input__field"
+                    data-validate="{ required: true }"
+                >
+            </div>
+            <div class="dashboard__input country_id _required">
+                <label class="label" for="country_id">
+                    <?= $block->escapeHtml(__('Country')) ?>
+                </label>
+                <div class="control select _required">
+                    <?php $countryValue = $company->getCountryId(); ?>
+                    <select
+                        name="country_id"
+                        id="country_id"
+                        class="select__field select__field--native"
+                        title="<?= $block->escapeHtml(__('Country')) ?>"
+                        data-validate="{ required: true }"
+                    >
+                        <option value=""></option>
+                        <?php foreach($countries as $countryCode => $country): ?>
+                            <option value="<?= $countryCode ?>"
+                                <?= ($countryCode == $countryValue) ? ' selected' : '' ;?>>
+                                <?= $country ?>
+                            </option>
+                        <?php endforeach ?>
+                    </select>
+                </div>
+            </div>
+            <div class="dashboard__input region-id required">
+                <label class="label" for="region_id">
+                    <?= $block->escapeHtml(__('State/Province')) ?>
+                </label>
+                <div class="control select _required">
+                    <select
+                        name="region_id"
+                        id="region_id"
+                        class="select__field select__field--native"
+                        data-validate="{ required: true }"
+                    >
+                        <option value="">
```

Comment:
Add `disabled selected` attributes

Reply:
why? this is copy/past from magento core, and there are no `selected` `disabled` options